packages/platform-android/src/commands/runAndroid/runOnAllDevices.ts

```diff
@@ -54,19 +53,21 @@ function runOnAllDevices(
   }
   const devices = adb.getDevices(adbPath);
 
-  (devices.length > 0 ? devices : [undefined]).forEach(device => {
-    tryRunAdbReverse(args.port, device);
-    tryLaunchAppOnDevice(
-      device,
-      packageNameWithSuffix,
-      packageName,
-      adbPath,
-      args.mainActivity,
-    );
-  });
+  (devices.length > 0 ? devices : [undefined]).forEach(
+    (device: string | void) => {
+      tryRunAdbReverse(args.port, device);
+      tryLaunchAppOnDevice(
+        device,
+        packageNameWithSuffix,
+        packageName,
+        adbPath,
+        args.mainActivity,
+      );
+    },
+  );
 }
 
-function createInstallError(error) {
+function createInstallError(error: any) {
```

Comment:
```suggestion
function createInstallError(error: unknown) {
```

Reply:
This ain't working, but I have got a different solution.